src/deroff.rs

```diff
@@ -1,10 +1,40 @@
 /// A translation of https://github.com/fish-shell/fish-shell/blob/e7bfd1d71ca54df726a4f1ea14bd6b0957b75752/share/tools/deroff.py
 // """ Deroff.py, ported to Python from the venerable deroff.c """
+extern crate regex;
+use regex::Regex;
 
 // class Deroffer:
-struct Deroffer {}
+struct Deroffer {
```

Comment:
Instead of a struct for these regexes, could we just make all of them global statics?

Reply:
In the interest of maintaining parity with the Python implementation, perhaps we could leave as-is right now? I have a sense that some of the regexes may turn out not to be necessary down the road...